android/services/mongodb-remote-perftests/src/androidTest/java/com/mongodb/stitch/android/services/mongodb/remote/SyncPerformanceTestContext.kt

```diff
@@ -112,15 +113,18 @@ class SyncPerformanceTestContext(private val harness: SyncPerformanceIntTestsHar
             Tasks.await(testColl.deleteMany(Document()))
         }
 
-        if (::testMongoClient.isInitialized) {
+        // TODO: The test should be set up in such a way that each setup sets up an embedded mongo
```

Comment:
I can create a ticket for this, but I think this should really be a consequence of just doing general QoL improvements to the performance test harness